### ISSUE TITLE
Elevated urinary 2-hydroxybutyric acid

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6011]
**Preferred term label:** Elevated urinary 2-hydroxybutyric acid

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of 2-Hydroxybutyric acid in the urine is above the normal range.

**Parent term (use hpo.jax.org/app)** Elevated urinary carboxylic acid HP:0040156

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
